native/src/seal/util/rlwe.cpp

```diff
@@ -229,13 +231,14 @@ namespace seal
             // where e[j] <-- chi, u <-- R_3
 
             // Create a PRNG; u and the noise/error share the same PRNG
-            auto prng = parms.random_generator()->create();
+            auto prng = seed.has_value() ? parms.random_generator()->create(seed.value())
```

Comment:
This is only for your debugging right? This should not be long lived.

Reply:
I assumed it didn't hurt to leave it in; by all the defaults the random seed is used. You'd have to specifically try to use a seed. If you don't want me to commit this I can remove it. But it is the whole point of the PR.

Reply:
I've removed this statement back to the random number in the merge so if someone wants to debug with a known seed, they are set up to but it is not active.

Reply:
Also note that the seed was only used if it was provided, otherwise you would get the unseeded random number generator.